rmgpy/data/kinetics/library.py

```diff
@@ -434,6 +435,16 @@ def load(self, path, local_context=None, global_context=None):
             # Create a new reaction per entry
             rxn = entry.item
             rxn_string = entry.label
+
+            # Convert coverage dependence label to species label
+            if isinstance(entry.data, (SurfaceArrhenius, StickingCoefficient, SurfaceArrheniusBEP, StickingCoefficientBEP)):
+                if entry.data.coverage_dependence:
+                    coverage_dependence = {}
+                    entry.data.coverage_dependence = coverage_dependence
+                    for key, value in entry.data.coverage_dependence.items():
```

Comment:
again, it looks like you're iterating through an empty dictionary that you just made?